format-specs/geoparquet.md

```diff
@@ -159,6 +168,10 @@ This attribute indicates how to interpret the edges of the geometries: whether t
 
 If no value is set, the default value to assume is 'planar'.
 
+Note if edges is spherical then it is recommended that `orientation` is always set to 'counterclockwise'. If it is not set 
+then implementations should choose the smaller polygon for interpretation (but note this can only work with polygons 
```

Comment:
Note I don't entirely understand this so was just trying to represent what @mentin said. If anyone (@rouault? @jorisvandenbossche?) could review / suggest changes that'd be great. But I also want to cut this release soon, and this feels like a pretty minor point, so we can also just come back and fix it in a 0.4)

Reply:
Personally, I don't really like the "choose the smaller polygon" approach. How does one compare the two polygons, with what error margin? It feels error-prone and as mentioned means we cannot have polygons larger than the hemisphere. How about for `orientation: spherical` the reader just assumes `orientation: counterclockwise`?

Reply:
I think when this field is absent, the reader can make the most pragmatic choice for the reader, and we probably shouldn't make a recommendation about what that means here.  I think from a spec standpoint, having `edges: spherical` and lacking `orientation` gives the reader permission to reject these datasets outright instead of making any assumptions, if they so choose.  Or they can make their own stated assumptions about what to do when it is absent, but that is documentation on the reader side rather than the spec.

Reply:
Sounds good to me